src/plugins/persistence/couch/CouchObjectProvider.js

```diff
@@ -223,13 +223,17 @@ class CouchObjectProvider {
       // Network error, CouchDB unreachable.
```

Comment:
Should we just add a short-circuit for `AbortError`s here?

```js
if (error.name === 'AbortError') {
    return;
}
```

Reply:
yeah that makes more sense. thanks!